src/main/java/com/synopsys/integration/alert/web/security/authentication/saml/AlertWebSSOProfileOptions.java

```diff
@@ -44,7 +43,7 @@ public Boolean getForceAuthN() {
         try {
             final ConfigurationModel currentConfiguration = samlContext.getCurrentConfiguration();
             return samlContext.getFieldValueBoolean(currentConfiguration, AuthenticationDescriptor.KEY_SAML_FORCE_AUTH);
-        } catch (final AlertDatabaseConstraintException | AlertLDAPConfigurationException e) {
+        } catch (final AlertException e) {
             logger.error("Could not get the SAML force AuthN.", e);
```

Comment:
Since this line was exposed by this PR, can we make it use a real word? "Authentication"?

Reply:
Didn't even notice that. Good catch